src/timeseriesinsights/azext_timeseriesinsights/azext_metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+    "azext.isPreview": false,
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.0.67",
```

Comment:
We support `isExperimental` from `2.3.0`. And as there is a login bug with `2.3.0`, it's better to user `2.3.1` as the `minCliCoreVersion` here.

Reply:
```suggestion
    "azext.minCliCoreVersion": "2.3.1",
```